test/verifiedTests/base/testSolvers/testVerifyCobraProblem.m

```diff
@@ -0,0 +1,242 @@
+% The COBRAToolbox: testVerifyCobraProblem.m
+%
+% Purpose:
+%     - testVerifyCobraProblem tests the verifyCobraProblem method
+%
+% Author:
+%     - Thomas Pfau, April 2018
+%
+% Note:
+%       
+
+global CBTDIR
+
+%Start with an empty struct
+problem = struct();
+% create a valid problem:
+problem.A = [1,0;0,1];
+problem.b = [0;0];
+problem.csense = ['EE']';
+problem.lb = [-10;-10];
+problem.ub = [10;10];
+problem.osense = -1;
+problem.c = [0;1];
+res = verifyCobraProblem(problem);
+assert(res == 1);
+% remove A
+problem = rmfield(problem,'A');
+res = verifyCobraProblem(problem);
+assert(res == -1);
+%Add invalid A
+problem.A = [1,0;NaN,1];
+res = verifyCobraProblem(problem);
```

Comment:
Instead of testing for `== -1`, would it make sense to test for the error itself?

Reply:
if it would be thrown as an error: sure. 
But since It's just a display output, not really, except having loads of "diary" commands, which I would like to avoid.

Reply:
Yes, I meant to actually through an error ... Maybe a `soft` and a `hard` mode?

Reply:
I changed them to warnings, which can be validated via lastwarn. 
A hard mode is not really an option, as it could break existing code that relies on a soft return value instead of an error.